tests/prague/eip7692_eof_v1/eip6206_jumpf/test_jumpf_execution.py

```diff
@@ -149,3 +151,193 @@ def test_callf_to_non_returning_section(
             validity_error=EOFException.MISSING_STOP_OPCODE,
         ),
     )
+
+
+def test_jumpf_stack_size_1024(
+    eof_state_test: EOFStateTestFiller,
+):
+    """Test stack reaching 1024 items in target function of JUMPF"""
+    eof_state_test(
+        data=Container(
+            sections=[
+                Section.Code(
+                    code=Op.PUSH0 * 1022 + Op.JUMPF[1],
+                    max_stack_height=1022,
+                ),
+                Section.Code(
+                    Op.SSTORE(slot_code_worked, value_code_worked) + Op.STOP,
+                    code_inputs=0,
+                    code_outputs=NON_RETURNING_SECTION,
+                    max_stack_height=2,
+                ),
+            ],
+        ),
+        container_post=Account(storage={slot_code_worked: value_code_worked}),
+    )
+
+
+def test_jumpf_with_inputs_stack_size_1024(
+    eof_state_test: EOFStateTestFiller,
+):
+    """Test stack reaching 1024 items in target function of JUMPF with inputs"""
+    eof_state_test(
+        data=Container(
+            sections=[
+                Section.Code(
+                    code=Op.PUSH0 * 1022 + Op.JUMPF[1],
+                    max_stack_height=1022,
+                ),
+                Section.Code(
+                    Op.SSTORE(slot_code_worked, value_code_worked) + Op.STOP,
+                    code_inputs=3,
+                    code_outputs=NON_RETURNING_SECTION,
+                    max_stack_height=5,
+                ),
+            ],
+        ),
+        container_post=Account(storage={slot_code_worked: value_code_worked}),
+    )
+
+
+def test_jumpf_stack_size_1024_at_push(
+    eof_state_test: EOFStateTestFiller,
+):
+    """Test stack reaching 1024 items in JUMPF target function at PUSH0 instruction"""
+    eof_state_test(
+        data=Container(
+            sections=[
+                Section.Code(
+                    code=Op.PUSH0 * 1023
+                    + Op.CALLF[1]
+                    + Op.POP * 1023
+                    + Op.SSTORE(slot_code_worked, value_code_worked)
+                    + Op.RETURN(0, 0),
+                    max_stack_height=1023,
+                ),
+                Section.Code(
+                    # stack has 1023 items
+                    Op.JUMPF[2],
+                    code_inputs=0,
+                    code_outputs=0,
+                    max_stack_height=0,
+                ),
+                Section.Code(
+                    Op.PUSH0 +
+                    # stack has 1024 items
+                    Op.POP + Op.RETF,
+                    code_inputs=0,
+                    code_outputs=0,
+                    max_stack_height=1,
+                ),
```

Comment:
Looks easily parapeterizable.

Reply:
> I think we are missing coverage in this area, e.g. what if instead of unconditionally jumping into the next section, section one could also `STOP`, `REVERT`, or `RETF` itself.

not sure I understant what you intend to verify with this exactly?
(Here RETF returns control back to section 0 and execution ends with RETURN always. The idea of this case is for stack to grow to max size (1024) inside JUMPF target)

Reply:
I think I was unclear, this test is completely fine but I meant to say that we should write more tests using this pattern on another PR. I'll open an issue and a PR to try to tackle this.